DasharoPayloadPkg/DasharoPayloadPkg.dsc

```diff
@@ -540,6 +540,7 @@
   gEfiMdeModulePkgTokenSpaceGuid.PcdPciSerialParameters|$(PCI_SERIAL_PARAMETERS)
 
   gUefiCpuPkgTokenSpaceGuid.PcdCpuMaxLogicalProcessorNumber|$(MAX_LOGICAL_PROCESSORS)
+  gUefiCpuPkgTokenSpaceGuid.PcdFirstTimeWakeUpAPsBySipi|FALSE

```

Comment:
@miczyg1 shouldn't this go to `PcdsFixedAtBuild`? 
building qemu fails with:

```
build.py...
 : error 3000: Type [PatchableInModule] of PCD [gUefiCpuPkgTokenSpaceGuid.PcdFirstTimeWakeUpAPsBySipi] in DSC file doesn't match the type [FixedAtBuild] defined in DEC file.
    
 

- Failed -
Build end time: 16:09:22, Sep.26 2024
Build total time: 00:00:08

Failed!
make[1]: *** [Makefile:526: /home/coreboot/coreboot/payloads/external/edk2/workspace/Build/DasharoPayloadPkgX64/RELEASE_COREBOOT/FV/UEFIPAYLOAD.fd] Error 1
make: *** [payloads/external/Makefile.mk:171: build/UEFIPAYLOAD.fd] Error 2
```

Reply:
I amended the commit to get https://github.com/Dasharo/edk2/commit/6b645e469f2e8d30be6823c1fb96b2a4427205cd.  Should have tested the build before approving.